hack/build.sh

```diff
@@ -42,5 +42,11 @@ if (echo "${TAGS}" | grep -q 'libvirt')
 then
 	export CGO_ENABLED=1
 fi
+if test "$(go env GOARCH)" = "arm64"
+then
+	# https://github.com/golang/go/issues/40492
+	LDFLAGS="${LDFLAGS} -linkmode external"
+	export CGO_ENABLED=1
```

Comment:
Why does CGO need to be enabled? I am also a bit confused as to why linkmode comes into play at all since the installer only use CGO when enabling libvirt support.

Reply:
As described in the golang bug mentioned in the commit message, the golang internal linker does not know how to link a binary as large as the installer for the linux/arm64 target.  Forcing use of the external linker is the only known workaround at the moment until that gets fixed.